examples/SingleParticleTest/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -93,4 +93,10 @@ namespace picongpu
     // = 1/c^2
     const float_X MUE0_EPS0 = float_X(1. / SPEED_OF_LIGHT / SPEED_OF_LIGHT);
 
+    /* Atomic unit of electric field in PIC Efield units */
+    const float_X ATOMIC_UNIT_EFIELD = float_X(SI::ATOMIC_UNIT_EFIELD / UNIT_EFIELD);
+
+    /* Atomic unit of time in PIC units */
+    const float_X ATOMIC_UNIT_TIME = float_X(SI::ATOMIC_UNIT_TIME / UNIT_TIME);
```

Comment:
*whispers to @ax3l to have a look*
... although I'm quite positive that this is okay